test/src/uitest.coffee

```diff
@@ -978,7 +978,7 @@ asyncTest 'Controller: Session switch test', ->
       equal editor.paletteWrapper.style.left is '0px', true, 'Using blocks'
       editor.setEditorState(false)
 
-      equal editor.paletteWrapper.style.left is '0px', false, 'No longer using blocks'
+      equal editor.paletteWrapper.style.left is '0px', true, 'No longer using blocks'
```

Comment:
Should `editor.setEditorState(false)` have modified the paletteWrapper metrics?

Reply:
Ah I see, because it slides of the screen to the right instead of the left. Should we update the test to check the `.left` of the main block area?

Reply:
Fixed the tests.